### DISCUSSION TITLE
dotnet in linux

### DISCUSSION THREAD

**Discussion by SS7D** [perpetualKid/FreeTrainSimulator#107] [Q&A]
I run it in centOS，but it show that：Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET program, but dotnet-multiplayerserver.dll does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.

**Comment by perpetualKid** [comment]
I'm trying to understand what you'd like to achieve: Do you want to run the multiplayer server under CentOS, or are you aiming to run the OR simulator? Fo the latter, there is some guidance in our wiki how to run OR under Linux using Wine https://github.com/perpetualKid/ORTS-MG/wiki/Linux-Wine .
For Multiplayer Server, also check the binary dll is normally MultiPlayerServer.dll, so depending where you have the OR folder, your command line would be like "dotnet MultiPlayerServer.dll 50000", may need to prepend the correct path for your case.
